udmis/bin/build

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+while [[ $# -gt 0 ]]; do
+  if [[ "$1" == "check" ]]; then
+    check=check
+    shift
+  elif [[ "$1" == "covg" ]]; then
+    covg=jacocoTestReport
+    shift
+  else
+    break
+  fi
+done
+
+ROOT=$(dirname $0)/..
+BASE=..
+cd $ROOT
+
+jarfile=build/libs/udmis-1.0-SNAPSHOT-all.jar
+
+newest=$(ls -rt `find src/ $BASE/gencode/java -type f` | tail -n 1)
```

Comment:
it's newest, the -r reverses the order. I suppose it would be the same as "ls -t | head -n 1" -- but I always did it the other way since I usually use the same at the command line (easier to see the last line output than the first)